core/domain/exp_domain.py

```diff
@@ -2331,6 +2331,31 @@ def _convert_states_v29_dict_to_v30_dict(cls, states_dict):
                 del answer_group['tagged_misconception_id']
         return states_dict
 
+    @classmethod
+    def _convert_states_v30_dict_to_v31_dict(cls, states_dict):
+        """Converts from version 30 to 31. Version 31 updates
+        the Voiceover duration to have duration of the audio file
+        under FileModel.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        for state_dict in states_dict.values():
+            voiceovers_mapping = (state_dict['recorded_voiceovers']
+                                  ['voiceovers_mapping'])
+            for voiceover in voiceovers_mapping.keys():
+                for content in voiceovers_mapping[voiceover].keys():
```

Comment:
` voiceovers_mapping[voiceover].keys` are going to be the langauge_code, right?

Reply:
Yes that is the case.

Reply:
I have changed this to match the 'language_code'.

Reply:
done.